dev/NavigationView/NavigationView.cpp

```diff
@@ -86,6 +82,12 @@ NavigationView::~NavigationView()
     UnhookEventsAndClearFields(true);
 }
 
+// IUIElement / IUIElementOverridesHelper
+winrt::AutomationPeer NavigationView::OnCreateAutomationPeer()
+{
+    return winrt::make<NavigationViewAutomationPeer>(*this);
+}
+
 void NavigationView::UnhookEventsAndClearFields(bool isFromDestructor)
```

Comment:
isFromDestructor seems unused

Reply:
Its default value is false and the destructor passes true.

---
In reply to: [352734345](https://github.com/microsoft/microsoft-ui-xaml/pull/1683#discussion_r352734345) [](ancestors = 352734345)

Reply:
Made an update where now it is being used for a specific revoker

---
In reply to: [352734345](https://github.com/microsoft/microsoft-ui-xaml/pull/1683#discussion_r352734345) [](ancestors = 352734345)